apps/web/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "NODE_ENV='production' next build",
```

Comment:
This should come from the environment instead of being hardcoded here

Reply:
I have the production docker file in another PR. I'll rebase when merge this one and remove from here.